jdk-wrapper.sh

```diff
@@ -189,6 +189,10 @@ extension="tar.gz"
 if [ "${JDKW_PLATFORM}" = "macosx-x64" ]; then
   extension="dmg"
 fi
+# JDK 6 and before use .bin extension, note no specific macosx download available for these versions (using linux)
```

Comment:
There are 2 options for download, a regular `.bin` and an rpm-wrapped `.bin`. I've updated the comment to mention the optionally wrapped rpm as another supported option for the `.bin` extension download.